_sass/base.scss

```diff
@@ -143,7 +143,7 @@ hr {
 }
 
 .banner {
-    width: 33%;
+    width: 60%;
```

Comment:
Is there a reason you changed this back to 60%?

Reply:
Now that you mention it, this should actually be even bigger. This number will only affect it on mobile, since the media queries below will come into effect on bigger screens.

Reply:
Wait, no. 60% *is* what I want. It was 33% that was too small on mobile.